### ISSUE TITLE
Battery level

### ISSUE THREAD

**Issue by PianSom** [mark-wade/homebridge-hive#8]
Hi - I'm not sure your battery code is working properly just yet. I updated to 1.4.0 and restarted homebridge. I then went to my ios home app, and looked at the details for my two thermostats. In both cases I got a "Fetching data from Hive API" on the homebridge log.

But in both cases the ios app reports 100% battery (under Details for the thermostat). This seems unlikely, since my batteries are some 4 months old, and both the Hive web page and Hive ios app show graphics of both sets of batteries being less than full.

**Comment by mark-wade** [comment]
Hm... it seems to be working for me, I wonder if there is something wrong with multiple thermostats.

Could you get the output of `/nodes` and email it to me (m.wade@me.com)?

To do that, find this line:

```
var successHandler = function(body){
```

And add below:

```
console.log(body);
```

**Comment by PianSom** [comment]
sent

**Comment by mark-wade** [comment]
Try with 1.4.1 just published and let me know if the issue is resolved

**Comment by PianSom** [comment]
Both thermostats showing 80% now. Which is credible.

Though perhaps slightly out of kilter with the graphic on the ios app ...

![img_2780](https://cloud.githubusercontent.com/assets/25636191/25129052/4f332f3a-2434-11e7-9569-7c40f0af5be1.PNG)

**Comment by mark-wade** [comment]
I think Hive's app uses that icon for 80 or below - mine is on 75% and looks like it's using the same icon.

**Comment by PianSom** [comment]
Hmm - odd.

I stopped homebridge and ran a sudo npm update -g homebridge-hive and then a sudo update-g homebridge (as I was out of date on 0.4.17).

I then tested and reported as above.

But I have just done a npm -g outdated and am seeing that I am still running homebridge-hive 1.4.0 and that 1.4.1 is available. I tried another update of homebridge-hive but it seems to have no effect. I'm not sure what is going on or what version I am actually running.

It may be worth logging the homebridge-hive version being run at startup
